### DISCUSSION TITLE
Using with Tenancy for Laravel, uploads fine, but doesn't show images in preview.

### DISCUSSION THREAD

**Discussion by jeremiahsherrill** [oneduo/nova-file-manager#270] [Q&A]
https://tenancyforlaravel.com/

Image uploads to the correct tenant folder, but when I go to the file manager the image reference is there, but not the image itself.   Not sure what I am missing.   

images are uploaded to  storage/{{tenant}}/app/public

Just wondering if there is something in the config I am missing.

**Comment by Rezrazi** [comment]
Hi

Without a simple reproduction we can't really understand what the issue is, have you tried https://oneduo.github.io/nova-file-manager/access-control.html#on-demand-filesystem
